tools/lib/capitalization.py

```diff
@@ -125,6 +125,10 @@
     r"he/him",
     r"she/her",
     r"they/them",
+    # Used to show user activity status
+    r"an hour ago",
+    r"more than 2 weeks ago",
+    r"yesterday",
```

Comment:
It's probably a bug to add these here. What we probably should be doing is tagging the entire string for translation: "Active an hour ago". Because different languages have different word orders and might need to conjugate words differently for different quantities, I don't think it's ever correct to have this sort of fragment. (The previous logic might have been correct in that `Last active: Yesterday` is two more or less independent sentence fragments, in a way that `Active yesterday` is not.

See https://zulip.readthedocs.io/en/latest/translating/internationalization.html for more background.